### PR TITLE
[Order] Remove fluent interfaces

### PR DESCRIPTION
**Pull request by HerbMat**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | no 
| License       | MIT

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
* Remove all obvious comments as this one - https://github.com/HerbMat/Sylius/blob/remove-fluent-interfaces-in-order/src/Sylius/Component/Order/Model/Adjustment.php#L15
* https://github.com/HerbMat/Sylius/blob/remove-fluent-interfaces-in-order/src/Sylius/Component/Order/Model/Comment.php#L22 - and this
* https://github.com/HerbMat/Sylius/blob/remove-fluent-interfaces-in-order/src/Sylius/Component/Order/Model/CommentAwareInterface.php#L26 - and this and all others

File: src/Sylius/Component/Order/Model/Order.php [hunk 1]
```diff
@@ -270,8 +258,6 @@ public function removeItem(OrderItemInterface $item)
             $item->setOrder(null);
```
Comment (stloyd) [inline src/Sylius/Component/Order/Model/Order.php]: This should be removed as item __must__ be assigned to some order, otherwise it's broken.

File: src/Sylius/Component/Order/Model/Order.php [hunk 2]
```diff
@@ -344,8 +326,6 @@ public function addAdjustment(AdjustmentInterface $adjustment)
             $adjustment->setAdjustable($this);
```
Comment (stloyd) [inline src/Sylius/Component/Order/Model/Order.php]: This should be removed as item __must__ be assigned to some "parent", otherwise it's broken.

File: src/Sylius/Component/Order/Model/Order.php [hunk 3]
```diff
@@ -357,8 +337,6 @@ public function removeAdjustment(AdjustmentInterface $adjustment)
             $adjustment->setAdjustable(null);
```
Comment (stloyd) [inline src/Sylius/Component/Order/Model/Order.php]: This should be removed as item __must__ be assigned to some "parent", otherwise it's broken.

File: src/Sylius/Component/Order/Model/Order.php [hunk 4]
```diff
@@ -458,8 +428,6 @@ public function removeComment(CommentInterface $comment)
             $comment->setOrder(null);
```
Comment (stloyd) [inline src/Sylius/Component/Order/Model/Order.php]: This should be removed as item __must__ be assigned to some order, otherwise it's broken.

File: src/Sylius/Component/Order/Model/Order.php [hunk 5]
```diff
@@ -650,7 +604,5 @@ public function removeIdentity(IdentityInterface $identity)
             $identity->setOrder(null);
```
Comment (stloyd) [inline src/Sylius/Component/Order/Model/Order.php]: This should be removed as identity __must__ be assigned to some order, otherwise it's broken.

File: src/Sylius/Component/Order/Model/OrderInterface.php [hunk 6]
```diff
@@ -179,7 +179,6 @@ public function setState($state);
      * Add an identity to this order.  Eg. external identity to refer to an ebay order id
```
Comment (stloyd) [inline src/Sylius/Component/Order/Model/OrderInterface.php]: Please remove double space and add dot at end.

File: src/Sylius/Component/Order/Model/OrderItem.php [hunk 7]
```diff
@@ -190,8 +182,6 @@ public function removeAdjustment(AdjustmentInterface $adjustment)
             $adjustment->setAdjustable(null);
```
Comment (stloyd) [inline src/Sylius/Component/Order/Model/OrderItem.php]: Same as with order adjustments.

File: src/Sylius/Component/Order/Model/Order.php (hunk 1 above)
Comment (michalmarcinkowski) [inline src/Sylius/Component/Order/Model/Order.php]: These changes are not related to this PR and that change should be discussed separately.

**Comment by michalmarcinkowski** [comment]
:+1:
